EpiAware/src/EpiInference/manypathfinder.jl

```diff
@@ -1,3 +1,15 @@
+"""
+A variational inference method that runs `manypathfinder`.
+"""
+@kwdef struct ManyPathfinderMethod <: AbstractEpiAwareOptMethod
```

Comment:
Do we want to have method in the name or call it ManyPathfinder (I know I know)

Reply:
I think either have `Method` postfix or have `Opt` postfix which I'm using to indicate optimization methods (including VI).

Reply:
And that isn't duplicating the type-information? I fine with doing this so more asking about good practice as in R that would be probably considered duplicated info.

Reply:
Thats a good point.